lobby/src/main/java/org/triplea/lobby/server/db/BadWordController.java

```diff
@@ -31,17 +29,27 @@ public void addBadWord(final String word) {
   }
 
   @Override
-  public List<String> list() {
-    final String sql = "select word from bad_words";
+  public boolean containsBadWord(final String testString) {
+    if (testString.isEmpty()) {
+      return false;
+    }
+
+    // Query to count if at least one bad word value is contained in the testString.
+    final String sql =
+        "select count(bw.word) "
+            + "from bad_words bw "
+            + "where ? like '%' || lower(bw.word) || '%' "
```

Comment:
To avoid differences between sql lower and toLowerCase, would you mind replacing `?` with `lower(?)` and removing the toLowerCase call?

Reply:
I'm not aware there could be any difference there. 

Updated: f5bc14daf